composer.json

```diff
@@ -29,7 +29,8 @@
     },
     "require-dev": {
 
-        "phing/phing": "2.14.0"
+        "phing/phing": "2.14.0",
```

Comment:
The main issue here is that those settings (dev or no dev) are really only useful if the vendor directory is empty in the main repo (and then built for packages with no dev or for developers with dev). Since we are including the built vendor directory in the codebase, the goal is to never bring in anything that is not required for the production install (Scott and I decided to do this since there is not enough infrastructure and developer knowledge to make this leap yet; the goal was to end up there after modernization effort is complete though). I hope this makes sense.

If we had an instruction set on the composer wiki page to install phing globally, then that would be ideal, and then could remove this (and the awkward manually deleting of the package) from the composer package.